### ISSUE TITLE
Autocomplete overwriting dot character

### ISSUE THREAD

**Issue by Ja-sonYun** [yegappan/lsp#71]
<img width="302" alt="Screen Shot 2022-10-03 at 10 40 58" src="https://user-images.githubusercontent.com/46551097/193487545-f500cd99-a4bf-4f00-bded-b6be1dcf3ec8.png">
<img width="330" alt="Screen Shot 2022-10-03 at 10 41 05" src="https://user-images.githubusercontent.com/46551097/193487550-076f8411-ab6c-41cf-a905-188112cf4ed1.png">

While i using typescript language server and accessing to the properties, autocomplete is overwriting even a dot. But when i using rust-analyzer, it doesn't autocompleting like typescript lang server. It works properly.  
Seems like rust-analyzer's autocomplete is not activated with dot key input, but typescript does.

Temporary i'm adding below codes after [this block](https://github.com/yegappan/lsp/blob/a53978a37752d91945044e548828d0b05125a3bc/autoload/lsp/handlers.vim#L217-L224) and it seems like working properly with other language servers.
```vimrc
if matchstr(getline('.'), '\%' .. (col('.') - 1) .. 'c.') == '.'
  start_col += 1
endif
```

Or is there any option to handle this?

**Comment by yegappan** [comment]
I am not able to reproduce this issue using the typescript language server.

What is the value of the "iskeyword" option in the buffer containing the typescript file? Which typescript language server are you using?  Can you provide a simple typescript file to reproduce the problem?

**Comment by yegappan** [comment]
I am able to reproduce this issue now.  Will look into fixing this.

**Comment by yegappan** [comment]
I have pushed 2f23313004cccdabbc95ad6ffef6795cddff5c87 to address this.  Can you try the latest version of the plugin?

**Comment by techntools** [comment]
It is working. I had the same issue.

**Comment by yegappan** [comment]
Fixed by 2f23313004cccdabbc95ad6ffef6795cddff5c87
